txpool/account.go

```diff
@@ -172,6 +172,21 @@ func (a *account) setNonce(nonce uint64) {
 	atomic.StoreUint64(&a.nextNonce, nonce)
 }
 
+// Demotions returns the current value of demotions
+func (a *account) Demotions() uint64 {
+	return atomic.LoadUint64(&a.demotions)
+}
+
+// resetDemotions sets 0 to demotions to clear count
+func (a *account) resetDemotions() {
+	atomic.StoreUint64(&a.demotions, 0)
+}
+
+// incrementDemotions increments demotions
+func (a *account) incrementDemotions() {
+	atomic.AddUint64(&a.demotions, 1)
+}
+
```

Comment:
Any particular reason why you made this thread safe? IIRC, that field is never mutated by multiple gorutines.

Reply:
Fixed deb013150aef147eb22e47c4cfd2828d62524f81